docs/notebooks/ExB_drift.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>For v_y I think I've always seen this done with a charge-signed \omega_c, so then there's no need to add the q / |q| part - but this is nice and lucid! :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Yes, I figured that keeping \omega_c unsigned would make it consistent with the <code>formulary.parameters.gyrofrequency()</code> function in PlasmaPy. Feel free to modify any of the notation if you think it's appropriate. </p>